packages/relay/package.json

```diff
@@ -17,8 +17,7 @@
         "@helia/json": "^1.0.0",
         "@types/crypto-js": "^4.1.1",
         "@unirep-app/contracts": "1.0.0",
-        "@unirep/core": "2.0.0-beta-4",
-        "anondb": "^0.0.13",
```

Comment:
anondb should keep in relay packages

Reply:
Thank you for the feedback, it's been moved to "@unirep/core": "2.0.0-beta-6"